VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/participants-panel/participants-panel.component.spec.ts

```diff
@@ -844,4 +855,17 @@ describe('ParticipantsPanelComponent', () => {
         const updatedHandRaiseCount = component.participants.filter(x => x.hasHandRaised()).length;
         expect(updatedHandRaiseCount).toBe(0);
     });
+
+    it('should update participants', () => {
+        component.nonEndpointParticipants = [];
+        const mappedParticipants = mapper.mapFromParticipantUserResponseArray(participants);
+        participantPanelModelMapperSpy.mapFromParticipantUserResponseArray.and.returnValue(mappedParticipants);
```

Comment:
Same as above. Can't participantPanelModelMapperSpy.mapFromParticipantUserResponseArray.and.callThrough be used instead?